### DISCUSSION TITLE
Multi-user usage understanding

### DISCUSSION THREAD

**Discussion by HakanKoseoglu** [magicbug/Cloudlog#1010] [General]
I've got an installation with two users, myself and my wife - and I'd also like to set up an SWL user for 'heard' records. This way we both can import our own ADI files, then track our progress (or lack of!) on the grid maps and I can load ADI files from PSK Reporter too. 

So I've set up these three, using a single 'logbook'. Since there can be only one log-book, I was hoping to use the operator / logged user to be the key. It looks like it doesn't work like that and the 'station' is the main key. If I change the station, it changes for all users, logged in or not.

In a similar manner, the logbook and the grid maps etc. show contacts from all users, not just the user who has logged in. I noticed a commit (21342cf09fdb94bb72e36d3aa532bceb34dd9d0f) in the v2 branch which adds this to the log book get_qso function so the intention was there at some point - this doesn't exist in the master branch.

My wife hadn't made many contacts so far, being a newly minted amateur and hasn't used much yet so I didn't notice this until recently.

I realised I had a problem when I managed to import the wrong ADI file with thousands of contacts when I was trying to load them under the 'SWL' user and realised I was doing something different and my rather sparse grid map was suddenly significantly better! :)

So, obviously I'm using it wrong. Any recommendations on how to achieve what I'm looking for, if possible?

Cheers
Hakan M0KHR

**Comment by domnantas** [comment]
I'm trying to understand this too. Our club is considering hosting an instance of Cloudlog, but as far as I understand it's not going to work as we expect. We were hoping to have separate user accounts and multiple stations, each person having own logbook, contest stats etc + have the ability to view all club logs and stats. Cloudlog doesn't seem to operate this way - only one station can be active at a time.

I guess the solution is to host a personal instance of Clublog for each member of the club, but that way we won't be able to view all QSOs and stats in one place

**Reply by domnantas** [reply]
I see that User management updates are in high priority:
https://github.com/magicbug/Cloudlog/projects/8#card-52575011
https://github.com/magicbug/Cloudlog/discussions/789

Hopefully this will allow having separate logs for each user and a dashboard view where all user logs/stats are visible (and filterable)
